go_optouts/store/memory.py

```diff
@@ -1,9 +1,31 @@
-from zope.interface import implements
-from interface import IOptOutStore
 import uuid
 
+from zope.interface import implements
+
+from .interface import IOptOutBackend, IOptOutCollection
```

Comment:
No I was just wondering why the choice, from within the same package is a good enough reason for me.